bin/provision.sh

```diff
@@ -2,21 +2,32 @@
 # Run this from wptsync repo root
 set -euo pipefail
 
-# WPT_SSH_HGMO: path to private key; assuming pk is $WPT_SSH_HGMO.pub
-# WPT_SSH_GITHUB: path to private key; assuming pk is $WPT_SSH_GITHUB.pub
-# WPT_CREDENTIALS: path to production credentials ini
+# The "install credentials" step will be skipped unless all three of these
+# environment variables are set.
+#   WPT_SSH_HGMO: path to private key; assuming pk is $WPT_SSH_HGMO.pub
+#   WPT_SSH_GITHUB: path to private key; assuming pk is $WPT_SSH_GITHUB.pub
+#   WPT_CREDENTIALS: path to production credentials ini
 
 # _repo_root: path to wpt-sync repo
 # _image_name: name of docker image, tagged with HEAD sha
 # _tempdir: path to workspace for ansible
 
+if [ "$#" -ne 2 ]; then
+    echo Please specify a suitable git tag and message for this commit.
+    echo Usage: $0 \<tag\> \"\<message\>\"
+    echo The last tag associated with a docker build is $(git tag --list | grep .*-image | tail -1)
```

Comment:
fwiw this could be written `git tag --list '*-image' | tail -1`

Reply:
thanks! :)